background/redux-slices/ledger.ts

```diff
@@ -91,19 +93,27 @@ const ledgerSlice = createSlice({
     setDeviceConnectionStatus: (
       immerState,
       {
-        payload: { deviceID, status },
-      }: { payload: { deviceID: string; status: LedgerConnectionStatus } }
+        payload: { deviceID, status, isBlindSigner },
+      }: {
+        payload: {
+          deviceID: string
+          status: LedgerConnectionStatus
+          isBlindSigner?: boolean
+        }
+      }
     ) => {
       if (
         immerState.currentDeviceID === deviceID &&
         status === "disconnected"
       ) {
         immerState.currentDeviceID = null
       }
-
       const device = immerState.devices[deviceID]
       if (!device) return
       device.status = status
+
+      if (typeof isBlindSigner === "undefined") return
+      device.isBlindSigner = isBlindSigner
```

Comment:
Not relevant anymore per previous comment, but

```suggestion
      device.isBlindSigner = isBlindSigner ?? false
```